eahub/localgroups/views.py

```diff
@@ -94,6 +94,21 @@ def profile(self):
     def get_type(self):
         return "group"
 
+class SendGroupMessageView(SendMessageView):
+    def get_recipient(self):
+        return LocalGroup.objects.get(slug=self.kwargs["slug"], is_public=True)
+
+    # if the group doesn't have a set email address, emails are sent to the group organiser(s)
+    def get_recipient_email(self):
+        local_group = LocalGroup.objects.get(slug=self.kwargs["slug"])
+        if local_group.email == "":
+            return local_group.organisers_emails().split(" ")
```

Comment:
that probably won't work because the list is using ", " to separate emails

Reply:
you can call something as `[user.email for user in local_group.organisers.all()]`